lib/rules/no-pointer-down-event-binding.js

```diff
@@ -26,20 +27,20 @@ function isActionModifer(node) {
 }
 
 /**
- * Check if an event name is a "down" event
+ * Check if an event name is a pointer "down" event
  *
  * @param {string} eventName
  * @return {boolean}
  */
-function isDownEvent(eventName) {
-  return eventName.toLowerCase().endsWith('down');
+function isPointerDownEvent(eventName) {
```

Comment:
I think it's best that this util still returns a boolean like it used to, now it returns either an array or `null`.